src/test/resources/wrong_content.csv

```diff
@@ -0,0 +1,4 @@
+type,fruit,quantity,date
+banana
+banana
+apple,200,
```

Comment:
missing empty line in the end of the file

Reply:
fixed